src/asm/parser.y

```diff
@@ -1100,7 +1105,7 @@ nextu		: T_POP_NEXTU { sect_NextUnionMember(); }
 endu		: T_POP_ENDU { sect_EndUnion(); }
 ;
 
-ds		: T_POP_DS uconst { sect_Skip($2, true); }
+ds		: T_POP_DS uconst { sect_Skip(2, true); }
```

Comment:
I'd blame this mistake for the random tests failing.

Reply:
"Why is this number in hex for no good reason, let's change it." >_< Thanks.